docs/pages/ios-quickstart.md

```diff
@@ -0,0 +1,164 @@
+
+<pre><code class="swift">
+func bindStatus(withPublisher publisher: inout Published<`String`>.Publisher)
+</code></pre>
+
+<pre><code class="swift">
+class FormState: ObservableObject {
+	//...
+	@Published var usernameStatus: String = StatusCodes.shared.UNMODIFIED
+    @Published var passwordStatus: String = StatusCodes.shared.UNMODIFIED
+	
+	init() {
+        formModel.form
+            .field(id: FormModel.companion.USERNAME)?
+            .bindStatus(withPublisher: &$usernameStatus)
+   
+        formModel.form
+            .field(id: FormModel.companion.PASSWORD)?
+            .bindStatus(withPublisher: &$passwordStatus)
+	}
+}
+</code></pre>
+
+<p class="comment">Then we can customize the errorMessages that we want to show depending of the UI or wording that we are managing in any case</p>
+
+<pre><code class="swift">
+final class FormState: ObservableObject {
+	//...
+	@Published var usernameStatus: String = StatusCodes.shared.UNMODIFIED
+    @Published var passwordStatus: String = StatusCodes.shared.UNMODIFIED
+	
+	var usernameErrorMessage: String? {
+        switch usernameStatus {
+        case StatusCodes.shared.REQUIRED_UNSATISFIED:
+            return LocalizedString.FormState.nameRequiredError
+        default:
+            return nil
+        }
+  }
+  
+    var passwordErrorMessage: String? {
+        switch passwordStatus {
+        case StatusCodes.shared.MIN_LENGTH_UNSATISFIED:
+            return LocalizedString.FormState.passwordMinLengthError
+        case StatusCodes.shared.REQUIRED_UNSATISFIED:
+            return LocalizedString.FormState.passwordRequiredError
+        default:
+            return nil
+        }
+    }
+	//...
+}
+</code></pre>
+<p class="comment">As we can see in the above snippet, we are responding to the publisher statuses that we capture from the `StatusCodes` types that exist in the library source code.</p>
+
+<p>In the view we can implement this error responses in this way:</p>
+
+<pre><code class="swift">
+struct FormView: View {
+    @ObservedObject var formState: FormState
+	
+    var body: some View {
+        VStack {
+            TextField("Username textfield", text: formState.username)
+            if let usernameErrorMessage = formState.usernameErrorMessage {
+                ErrorMessageView(message: usernameErrorMessage)
+            }
+            SecureField("Password textfield", text: formState.password)
+            if let passwordErrorMessage = formState.passwordErrorMessage {
+                ErrorMessageView(message: passwordErrorMessage)
+            }
+        }
+    }
+}
+</code></pre>
+
+And that's one of the easiest forms we can create using **FlowForms**, we don't need to care about managing the field's validation triggering nor making any complex logic to enable or disable the continue button.
```

Comment:
I think we are missing the part of the enabling/disabling of the button based on the form's status, what do you think about adding it?